sickbeard/helpers.py

```diff
@@ -1409,7 +1409,11 @@ def get_size(start_path='.'):
     for dirpath, dirnames, filenames in ek.ek(os.walk, start_path):
         for f in filenames:
             fp = ek.ek(os.path.join, dirpath, f)
-            total_size += ek.ek(os.path.getsize, fp)
+            try:
+                total_size += ek.ek(os.path.getsize, fp)
+            except OSError, e:
```

Comment:
@ElmerLastdrager Try to use `except OSError as e:` instead, we are trying to write the new code to be as much possible compatible with python3 to make it easier to port one day.